backend/config/index.js

```diff
@@ -1,3 +1,11 @@
-const config = require('./variables.json');
+// const config = require('./variables.json');
+require('dotenv').config();
+
+const config = {
+  "MONGO_DB_URL": process.env.MONGODBURL,
+  "JWT_SECRET_KEY": process.env.JWTSECRETKEY,
+  "JWT_EXPIRES_IN" : process.env.JWTEXPIRESIN,
+  "baseURL" : process.env.BASEURL,
```

Comment:
please rename `baseURL` with `BASE_URL`

Reply:
Here, the baseURL is the variable used everywhere in the file. So, changing this would be required to change everywhere.
I can change the BASEURL to BASE_URL from .env file

Reply:
It's just the renaming task, so I think we can do it everywhere.